cfme/tests/configure/test_access_control.py

```diff
@@ -506,7 +513,9 @@ def test_group_crud(group_collection):
 @pytest.mark.sauce
 @pytest.mark.tier(2)
 @test_requirements.tag
-def test_group_crud_with_tag(provider, tag_value, group_collection):
+@pytest.mark.provider(classes=[VMwareProvider], selector=ONE)
+@pytest.mark.usefixtures('setup_provider')
```

Comment:
In such cases I would inherit fixture directly like
 ```
def test_group_crud_with_tag(appliance, provider, setup_provider,  tag_value, ):
```

Reply:
Will do.